frameworks/kafka/universe/config.json

```diff
@@ -324,15 +324,21 @@
         },
         "compression_type": {
           "title": "compression.type",
-          "description": "Specify the final compression type for a given topic. This configuration accepts the standard compression codecs ('gzip', 'snappy', lz4). It additionally accepts 'uncompressed' which is equivalent to no compression; and 'producer' which means retain the original compression codec set by the producer.",
+          "description": "Specify the final compression type for a given topic. This configuration accepts the standard compression codecs ('gzip', 'snappy', 'lz4'). It additionally accepts 'uncompressed' which is equivalent to no compression; and 'producer' which means retain the original compression codec set by the producer.",
           "type": "string",
           "default": "producer"
         },
         "delete_topic_enable": {
           "title": "delete.topic.enable",
           "description": "Enables delete topic. Delete topic through the admin tool will have no effect if this config is turned off",
           "type": "boolean",
-          "default": false
+          "default": true
```

Comment:
If i recall correctly, this change is also made in the Confluent mangle?

Reply:
That we set it to true?

Reply:
We do not appear to.

Reply:
OK, I may have been misremembering.